app/src/main/java/org/stepik/android/domain/base/PaginationDirection.kt

```diff
@@ -1,5 +1,5 @@
 package org.stepik.android.domain.base
 
 enum class PaginationDirection {
-    UP, DOWN
+    UP, DOWN, LEFT, RIGHT
```

Comment:
I don't think there is a real need to differ between `RIGHT` and `DONW` direction. It will make sense in case of grid that we can scroll in all 4 directions at once but here we always have only 2.

Reply:
Removed all directions and replaced with `PREV` and `NEXT`.